Front-end/store/src/app/order-page/order-page.component.html

```diff
@@ -0,0 +1,13 @@
+<div class="new-order-block">
+  <app-personal-data-form #dataForm [userData]="(userProfile$ | async)!"></app-personal-data-form>
+  <app-order-info [orderProducts]="basketProducts"></app-order-info>
+  <div class="order-bottom-block">
+    <div class="total-info">
+      <div class="address-info">Заказ будет ожидать вас по адресу: ХХХ</div>
+      <span class="total-cost">Итоговая сумма заказа: <span>{{totalCost}} &#8381;</span></span>
+    </div>
+    <div class="order-button-block">
```

Comment:
Can we swap places app-personal-data-form and app-order-info and move this block to personal data component?